src/App.vue

```diff
@@ -221,10 +221,16 @@ h3 {
   text-align: center;
 }
 
+div.threeColumns {
+  box-sizing: border-box;
+}
 div.statList {
-  width: 240px;
+  width: 32%;
   float: left;
-  padding: 10px;
+  padding: 5px;
+}
+div.clearFloat {
```

Comment:
mozno `div.statList:after` ? ale neviem ci to bude fungovat

Reply:
je mozne, ze tu triedu potom budeme chciet pouzit aj inde nez po statlistu.